domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientIT.java

```diff
@@ -67,7 +67,7 @@
     System.setProperty(LoggerFactory.LOGGER_DELEGATE_FACTORY_CLASS_NAME, "io.vertx.core.logging.Log4j2LogDelegateFactory");
   }
 
-  @BeforeClass
+ // @BeforeClass
```

Comment:
This comment-out is not related to this pull request and should go into a separate pull request to be discussed and reviewed separately.

Reply:
Sorry, it appeared here by mistake. Fixed